app/models/event_log.rb

```diff
@@ -109,19 +109,19 @@ def self.for(user)
     EventLog.order('created_at DESC').where(uid: user.uid)
   end
 
-private
-
-  def validate_event_mappable
-    unless entry
-      errors.add(:event_id, "must have a corresponding `LogEntry` for #{event_id}")
-    end
-  end
-
   def self.convert_ip_address_to_integer(ip_address_string)
```

Comment:
Do you think the author of these methods expected them to be private?

Reply:
Thanks for taking a look Erin.

I guess maybe, but as pure functions, the only effect I can think of in terms of being private would be to discourage use from outside this class. Ruby seems to have a `private_class_method` thing you can use, but I don't really know what it does. ..

Reply:
I don't think private class methods are particularly easy in ruby, and as you say there shouldn't be any side effects